training/js/__tests__/divide.test.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const divide = require('../divide');
+
+test('divide 12 / 4 to equal 3', () => {
+  expect(divide(12, 4)).toBe(3);
+});
```

Comment:
Try to configure your editor to add newlines to the end of file on save.

Reply:
Ah, did not realize before needed a default one, in the event there's no .editorconfig in the repo. Just created an .editorconfig matching the example file here http://editorconfig.org/ in the parent directory where I keep my code. Amended

Reply:
Nice! 👍